### PR TITLE
add carp suit to autodrobe

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
:fish:

## Why / Balance
:fish:

## Technical details
:fish:

## Media
:fish:
- [:fish:] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
:fish:

**Changelog**
:cl:
- tweak: Carp suit has been moved into the AutoDrobe.

### CODE REVIEW

**Comment by DEATHB4DEFEAT** [comment]
🐟

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/theater.yml [hunk 1]
```diff
@@ -73,6 +73,7 @@
     ClothingCostumeNaota: 1 # Nyano - Clothing addition
     ClothingUniformJumpsuitSober: 1 # DeltaV - Clothing addition
     ClothingHeadHatCatEars: 2 # DeltaV - Move cat ears out of syndicate uplink into the AutoDrobe
+    ClothingOuterSuitCarp: 2 # DeltaV - Move carp suit out of syndicate uplink into the AutoDrobe
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/theater.yml:76]: > Move carp suit out of syndicate uplink

?

Comment (deltanedas) [reply Resources/Prototypes/Catalog/VendingMachines/Inventories/theater.yml:76]: see #404

**Review by DEATHB4DEFEAT — APPROVED** [review]
🐟

**Comment by deltanedas** [comment]
:fish:

**Review by Colin-Tel — APPROVED** [review]
🐟

**Pull request merged**